src/libs/actions/TeachersUnite.js

```diff
@@ -0,0 +1,175 @@
+import _ from 'underscore';
+import Onyx from 'react-native-onyx';
+import lodashGet from 'lodash/get';
+import ONYXKEYS from '../../ONYXKEYS';
+import * as API from '../API';
+import Navigation from '../Navigation/Navigation';
+import CONST from '../../CONST';
+import * as ReportUtils from '../ReportUtils';
+import * as OptionsListUtils from '../OptionsListUtils';
+
+let sessionEmail = '';
+let sessionAccountID = 0;
+Onyx.connect({
+    key: ONYXKEYS.SESSION,
+    callback: (val) => {
+        sessionEmail = lodashGet(val, 'email', '');
+        sessionAccountID = lodashGet(val, 'accountID', 0);
+    },
+});
+
+let allPersonalDetails;
+Onyx.connect({
+    key: ONYXKEYS.PERSONAL_DETAILS_LIST,
+    callback: (val) => (allPersonalDetails = val),
+});
+
+/**
+ * @param {String} firstName
+ * @param {String} phoneOrEmail
+ * @param {String} [lastName]
```

Comment:
Sorry if this was not stated in the description, but first and last names should be required. Also, we should display an error if the last name is not set in the form.

```suggestion
 * @param {String} lastName
```

Reply:
Correct @dhairyasenjaliya

Reply:
Yes, let's re-use it